lib/commons/color/get-background-color.js

```diff
@@ -25,6 +25,18 @@ color.getBackgroundColor = function getBackgroundColor(
 		const clientHeight = elm.getBoundingClientRect().height;
 		const alignToTop = clientHeight - 2 >= window.innerHeight * 2;
 		elm.scrollIntoView(alignToTop);
+
+		// ensure element is scrolled into view horizontally
+		let center;
+		do {
+			const rect = elm.getBoundingClientRect();
+			const x = 'x' in rect ? rect.x : rect.left;
```

Comment:
A comment explaining when `x` wouldn't be in `rect` may be helpful for the next person.